test/integration/cloudant.integration.test.js

```diff
@@ -16,21 +16,19 @@
 
 'use strict';
 
+const assert = require('assert');
 const { CloudantV1 } = require('../../index.ts');
 const authHelper = require('../resources/auth-helper.js');
 
-// Use this to retrieve test-specific config properties from your credentials file.
```

Comment:
I think this is an unintentional removal.

Reply:
No, it was intentionally moved to line 26.